setup.py

```diff
@@ -513,6 +524,17 @@ def main(args):
         if len(args)>=4:
             minifier = args[3]
         install_html5(install_dir, minifier)
+        if len(args)<3:
```

Comment:
Why only if there are less than 3 args?
Shouldn't this be done in all cases?

Reply:
I recall I did it to preserve `default-settings.txt` inside `install_dir` if the installation directory is specified by user. This should keep the logic for `bdist` intact.